.github/workflows/rust.yml

```diff
@@ -25,8 +25,10 @@ jobs:
     - name: Build
       run: cargo build --verbose --workspace --exclude runqslower
     - name: Run tests
-      # Skip tests that require BTF built into kernel
+      # Skip BTF tests which require sudo 
       run: cargo test --verbose --workspace --exclude runqslower -- --skip test_object
```

Comment:
Do we still need the "--skip test_object" in this line?

Reply:
Yes.  the btf tests are all labeled test_object and require root permissions.  libbpf-rs/.cargo/config specifically turns on running the libbpf-rs tests with sudo.

If the config is pushed up into the root of the project, a "cargo clean" runs into permission errors.  I didn't want to look too deep into this issue